src/types.js

```diff
@@ -150,6 +150,11 @@ export type StoryChange = {
   linked_file_ids?: Array<ID>,
 };
 
+export type StorySearchResult = {
+  data: Array<Story>,
```

Comment:
I really want to call it `stories`, but at the same time, this is not how it's called in API.

Reply:
😅 

I think you'll see why this is Soon™! 😉